airbyte-integrations/bases/debezium-v1-9-6/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -95,14 +96,14 @@ protected Properties getDebeziumProperties() {
   }
 
   public static String getTableIncludelist(final ConfiguredAirbyteCatalog catalog) {
-    // Turn  "stream": {
-    //          "namespace": "schema1"
-    //          "name": "table1
-    //        },
-    //        "stream": {
-    //          "namespace": "schema2"
-    //          "name": "table2
-    //        } -------> info "schema1.table1, schema2.table2"
+    // Turn "stream": {
```

Comment:
The original is actually what I intended the comment to be, for readability reasons.
I manually reverted it back, which probably contributed to the build breaking.